docs/integrate/collectd/usage-collectd.md

```diff
@@ -6,60 +6,39 @@ so that collectd sends system metrics and CrateDB stores them.
 
 ## Prerequisites
 
-Docker runs all components consistently across Linux, macOS, and Windows.
-If you use Podman, substitute podman for docker in the commands.
+Use Docker or Podman to run all components. This approach works consistently
+across Linux, macOS, and Windows.
 
-### Commands
+### Files
 
-Prepare shortcut for the psql command.
+First, download and save all required files to your machine.
+- {download}`compose.yaml`
+- {download}`Dockerfile`
+- {download}`collectd-cratedb.conf`
 
-::::{tab-set}
-:sync-group: os
+### Services
 
-:::{tab-item} Linux and macOS
-:sync: unix
-To make the settings persistent, add them to your shell profile (`~/.profile`).
-```shell
-alias psql="docker run --rm -i --network=cratedb-demo docker.io/postgres:16 psql"
-```
-:::
-:::{tab-item} Windows PowerShell
-:sync: powershell
-To make the settings persistent, add them to your PowerShell profile (`$PROFILE`).
-```powershell
-function psql { docker run --rm -i --network=cratedb-demo docker.io/postgres:16 psql @args }
-```
-:::
-:::{tab-item} Windows Command
-:sync: dos
-```shell
-doskey psql=docker run --rm -i --network=cratedb-demo docker.io/postgres:16 psql $*
-```
-:::
-
-::::
+Start services using Docker Compose or Podman Compose.
+If you use Podman, replace `docker` with `podman` (or enable the podman‑docker
+compatibility shim) and run `podman compose up`.
 
-### CrateDB
-
-Create a shared network.
 ```shell
-docker network create cratedb-demo
+docker compose up
 ```
 
-Start CrateDB.
-```shell
-docker run --name=cratedb --rm -it --network=cratedb-demo \
-  --publish=4200:4200 --publish=5432:5432 \
-  --env=CRATE_HEAP_SIZE=2g docker.io/crate -Cdiscovery.type=single-node
-```
+To send the collected data to CrateDB, collectd is configured to load its
+[`postgresql` plugin].
 
-## Configure
+::::{dropdown} collectd configuration `collectd-cratedb.conf`
+:::{literalinclude} collectd-cratedb.conf
+:::
+::::
 
 ### Provision database
 
 Create a database table that stores collected metrics.
 ```shell
-psql "postgresql://crate:crate@cratedb:5432/" <<SQL
+docker compose run --rm --no-TTY psql psql "postgresql://crate:crate@cratedb:5432/" <<SQL
```

Comment:
`psql psql` reads like a typo to some. Maybe let's rename the first fragment (Docker Compose service name), which is at our disposal?